src/main/java/mantenimiento/codecounter/validators/logicalValidators/ControlStructureValidator.java

```diff
@@ -0,0 +1,43 @@
+package mantenimiento.codecounter.validators.logicalValidators;
+
+import mantenimiento.codecounter.interfaces.LogicalValidatorHandler;
+import mantenimiento.codecounter.constants.JavaRegextConstants;
```

Comment:
Te recomiendo utilizar importaciones estáticas para acceder directamente a lo que necesites
```suggestion
import static mantenimiento.codecounter.constants.JavaRegextConstants.FLOW_CONTROL_REGEX;
```
Y utilizarlo de la siguiente manera, por ejemplo: 

`lineOfCode.getFirst().matches(FLOW_CONTROL_REGEX)`